### ISSUE TITLE
"stockChart.indicators.values" array contains indicator object after click on close icon in indicator panel

### ISSUE THREAD

**Issue by mukeshbadgujar** [amcharts/amcharts5#507]
**Bug description**
after clicking on close icon for indicators those are plotting as a separate in panel, that are not removing from the array list in `stockChart.indicators.values`.

Steps to create issue.
1. Add Indicator in stock chart (MACD, Moving Average, Bollinger Bands etc)
2. do
```
        console.log(stockChart.indicators.values)
	console.log(stockChart.indicators.values.length)
```
3. MACD is added into new panel, and Moving Average, Bollinger Bands are added into same chart.
4. now click on Close icon of MACD
5. do
```
        console.log(stockChart.indicators.values)
	console.log(stockChart.indicators.values.length)
```
6. you will see, there is still 3 items in list. (means MACD is removed from FrontEnd but not from list in JS)

Example:
This code is freezing at site after clicking on show button, but it is working on local.
https://jsfiddle.net/6c7asf52/2/

**Environment **

amCharts version: am5 Latest version [5.2.14]
Browser and its version : Version 102.0.5005.61 (Official Build) (64-bit)
Related frameworks and their versions: vanilla JavaScript

**Comment by mukeshbadgujar** [comment]
The "stockChart.indicators.values" array will being empty only after you click on reset control.

**Comment by mukeshbadgujar** [comment]
@martynasma please check, waiting for your response.

**Comment by martynasma** [comment]
Fixed in [5.2.15](https://github.com/amcharts/amcharts5/releases/tag/5.2.15).

## [5.2.15] - 2022-08-06

### Added
- Added `PicturePattern` that allows using external images as patterns. [More info](https://www.amcharts.com/docs/v5/concepts/colors-gradients-and-patterns/patterns/#Image_patterns).

### Fixed
- Fixing issue where chart scrollbar would break when zooming.
- Fixing chart being too large and overflowing when zooming in ([Issue 471](https://github.com/amcharts/amcharts5/issues/471)).
- `pan:"zoom"` was not working on left-side Y Axis.
- Removing a panel from a `StockChart` was not removing indicator from charts `indicators` list.
- Zooming chart with custom timezone was resulting in errors in some cases.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
